writeengine/wrapper/we_colop.cpp

```diff
@@ -1533,12 +1542,16 @@ int ColumnOp::writeRow(Column& curCol, uint64_t totalRow, const RID* rowIdArray,
     char     charTmpBuf[8];
     uint64_t  emptyVal;
     int rc = NO_ERROR;
+    bool fillUpWEmptyVals = false;
+    bool fistRowInBlock = false;
```

Comment:
a typo worth fixing

Reply:
Fixed.

Reply:
Sorry, I meant 'fistRowInBlock' -> 'firstRowInBlock'.  :)

Reply:
I didn't miss the typo for the second time :)